src/containers/PerfherderGraphContainer/index.jsx

```diff
@@ -22,19 +23,38 @@ const styles = () => ({
 class PerfherderGraphContainer extends Component {
   state = {
     data: null,
+    tooltipModel: null,
+    canvas: null,
   };
 
   async componentDidMount() {
-    this.fetchSetData(this.props);
+    this.fetchSetData(this.props).then(() => {
+      const { handleTooltip } = this;
+
+      this.setState(prevState => {
+        const { options } = prevState;
+
+        options.tooltips.custom = function custom(tooltipModel) {
+          // eslint-disable-next-line no-underscore-dangle
+          handleTooltip(tooltipModel, this._chart.canvas);
```

Comment:
What I'm doing here is binding the environment of the chart (tooltipModel, canvas) with the environment of the `PerfherderGraphContainer` so I can mount/unmount `CustomTooltip` when ChartJS says that it should show up.